test/behaviour/connection/ConnectionStepsBase.java

```diff
@@ -68,6 +71,12 @@ void before() {
     }
 
     void after() {
+        // TODO: Remove this once the server segfault issue is fixed (grakn#6135)
+        try {
+            Thread.sleep(10);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
can we get rid of this now? or at least the TODO

Reply:
Umm I think this is still needed because there is an intermittent issue when closing the sessions at the end of each scenario where the server segfaults. The issue is NOT resolved, to the best of my knowledge.